apps/patient/src/utils/text.ts

```diff
@@ -156,6 +157,10 @@ export const text = {
     `We sent your ${isPlural ? 'prescriptions' : 'prescription'} to the pharmacy.`
 };
 
+function generatePhoneNumberLink(): string {
+  return `${text.questionVerb} <a href="tel:${text.questionsPhoneNumber}">${text.questionsPhoneNumber}</a>.`;
```

Comment:
can you change `tel:` to `sms:` and format the text so it's `sms:+15138663212`

Reply:
done